packages/testcontainers/src/port-forwarder/port-forwarder.ts

```diff
@@ -57,8 +54,9 @@ export class PortForwarderInstance {
       ? { container: 22, host: Number(process.env["TESTCONTAINERS_SSHD_PORT"]) }
       : 22;
 
+    const portForwarderId = new RandomUuid().nextUuid();
     const container = await new GenericContainer(SSHD_IMAGE)
-      .withName(`testcontainers-port-forwarder-${reaper.sessionId}`)
+      .withName(`testcontainers-port-forwarder-${portForwarderId}`)
       .withExposedPorts(containerPort)
       .withEnvironment({ PASSWORD: password })
       .withCommand([
```

Comment:
Alternatively, if a having a container name that starts with `testcontainers-port-forwarder-` is not important, removing the `.withName(...)` statement would have a similar effect: _When the Container engine creates the container instance, it will give it a random name that won't clash with other existing containers._

```suggestion
    const container = await new GenericContainer(SSHD_IMAGE)
      .withName(`testcontainers-port-forwarder-${reaper.sessionId}`)
      .withExposedPorts(containerPort)
      .withEnvironment({ PASSWORD: password })
      .withCommand([
```